### PR TITLE
Spans Processing and Exporting

### PR DESCRIPTION
**Pull request by beniamin**

### CODE REVIEW

**Comment by bobstrecansky** [comment]
While testing this out, I was using your (beautiful) Makefile.  I did notice that my package.lock got out of sync, but I was able to fix that by adding this stanza:

```
update:
	$(DC_RUN_PHP) composer update
```

Perhaps we should add that?

**Comment by beniamin** [comment]
> While testing this out, I was using your (beautiful) Makefile. I did notice that my package.lock got out of sync, but I was able to fix that by adding this stanza:
> 
> ```
> update:
> 	$(DC_RUN_PHP) composer update
> ```
> 
> Perhaps we should add that?

good ideea, I will add it in current PR.

**Comment by bobstrecansky** [comment]
By running `docker-compose up` followed by `make examples`, I was able to see a trace in zipkin:

image

**Comment by beniamin** [comment]
> By running `docker-compose up` followed by `make examples`, I was able to see a trace in zipkin:
> 
> image
> 
> We may want to include updates to the README.md so that new users know how to test this. Having a usable, testable bit can often help with momentum! I'm happy to put that together after we merge this PR.

done

**Comment by beniamin** [comment]
> While testing this out, I was using your (beautiful) Makefile. I did notice that my package.lock got out of sync, but I was able to fix that by adding this stanza:
> 
> ```
> update:
> 	$(DC_RUN_PHP) composer update
> ```
> 
> Perhaps we should add that?

done

File: Makefile [hunk 1]
```diff
@@ -2,6 +2,8 @@ DC_RUN_PHP = docker-compose run php
 
 install:
 	$(DC_RUN_PHP) composer install
+update:
+	$(DC_RUN_PHP) composer update
```
Comment (bobstrecansky) [inline Makefile]: :1st_place_medal:

File: README.md [hunk 2]
```diff
@@ -20,17 +20,19 @@ $ composer require open-telemetry/opentelemetry
 
 ### Examples
 
-You can use the [examples/AlwaysSampleTraceExample.php](https://github.com/open-telemetry/opentelemetry-php/tree/master/examples/AlwaysSampleTraceExample.php) file to test out the reference implementation we have.  This can be easily executed with docker by running `./resources/example-using-docker` from the root of the repository.
+You can use the [examples/AlwaysSampleTraceExample.php](https://github.com/open-telemetry/opentelemetry-php/tree/master/examples/AlwaysSampleTraceExample.php) file to test out the reference implementation we have.  
+First you should start zipkin server by running `docker-compose up -d` and then you can you can easily execute examples using `make example`.
```
Comment (bobstrecansky) [inline README.md]: Maybe this should read something along the lines of, "If you'd like to be able to view example traces, start the zipkin server....."?

**Pull request merged**